src/utils/service-requests/playlist-services.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+
+const getPlaylists = async (dataDispatch) => {
+  try {
+    const token = localStorage.getItem("token");
```

Comment:
store the localstorage token in a variable in your auth call and use it everywhere else rather than checking localstorage before every call

Reply:
Do you mean I should pass the token as an argument here in the functions?

Reply:
when you signup or login you get a token in response of those api calls. Save that token in a variable and use that everywhere else in your app. Rather than fetching it inside all functions